### PR TITLE
all components are case-sensitive

### PR DESCRIPTION
**Pull request by si458**
## Description
the components do not work correctly unless every first charactor is lowercase and the rest uppercase

## Specific Changes proposed
change first character to lowercase and rest uppercase

## Requirements Checklist
- [ ] Feature implemented / Bug fixed
- [ ] If necessary, more likely in a feature request than a bug fix
  - [ ] Unit Tests updated or fixed
  - [ ] Docs/guides updated
  - [ ] Example created ([starter template on JSBin](http://jsbin.com/axedog/edit?html,output))
- [ ] Reviewed by Two Core Contributors

all components are case-sensitive so first letter must be lowercase!

### CODE REVIEW

File: docs/guides/components.md [hunk 1]
```diff
@@ -35,30 +35,30 @@ The actual default component structure of the Video.js player looks something li
 
 ```
 Player
```
Comment (gkatsev) [inline docs/guides/components.md]: The idea for this section to go with the "class names" rather than the instance names.

**Comment by gkatsev** [comment]
I wonder if we should make a note that when you do `addChild` or `getChild` it should be camel-cased instead.
Also, whether those method should just be case-insensitive?
@videojs/core-committers should `addChild` and related methods be case-insensitive for component names?

**Comment by misteroneill** [comment]
I think they should be case-insensitive in the sense that they accept lowerCamelCase and UpperCamelCase because the latter matches the class names, but not UPPERCASE or lowercase.

**Comment by si458** [comment]
i think it be case-insensitive too :)

**Comment by gkatsev** [comment]
I created a new issue describing this and what we want to happen: https://github.com/videojs/video.js/issues/3436
So, going to close this PR. Thanks for your time @si458, if you feel up to it, would love it if you make a PR to fix #3436.

**Pull request closed without merging**